content/ADR-138-pick-realm-v2.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: adr
+adr: 138
```

Comment:
Please use ADR id same as PR (in this case 145) to avoid conflicts with other ADRs (there's already another one ongoing with same id).